satpy/readers/viirs_l1b.py

```diff
@@ -59,13 +60,14 @@ def platform_name(self):
             res = self.get('/attr/platform',
                            self.filename_info['platform_shortname'])
         except KeyError:
-            res = 'Suomi-NPP'
+            res = 'Unknown'
```

Comment:
I *think* this was to handle older versions of VIIRS L1B that didn't have a platform name. Hopefully those files shouldn't be used anymore. I'm ok with this change though.

Actually...we would only get a KeyError if the filename didn't have a `platform_shortname`. Should this try/except be removed and the `self.filename_info` be changed to `.get`?

Reply:
Eh leave this the way it is.